src/api/config.ts

```diff
@@ -48,3 +49,4 @@ export const rangerReconnectPeriod = (): number => Cryptobase.config.rangerRecon
 export const incrementalOrderBook = (): boolean => Cryptobase.config.incrementalOrderBook || false;
 export const isResizableGrid = ():boolean => Cryptobase.config.isResizable || false;
 export const isDraggableGrid = ():boolean => Cryptobase.config.isDraggable || false;
+export const languages = Cryptobase.config.languages || [];
```

Comment:
add check here if `Cryptobase.config.languages,length > 0`
and if it's empty return `['en']`